runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/RemoteExecutionTest.java

```diff
@@ -535,160 +532,6 @@ public void processElement(ProcessContext context) {
     }
   }
 
-  @Test
```

Comment:
Instead of creating a new file and copying the setup logic rename setup to `launchSdkHarness(PipelineOptions options)` and have each test run that method as the first line of each test. You'll also want to drop the `@Setup` annotation.